packages/api/src/hooks/useIsEuRegion.ts

```diff
@@ -0,0 +1,37 @@
+import { useMemo } from 'react';
+import useLandingCompany from './useLandingCompany';
+
+const useIsEuRegion = () => {
+    const { data: landing_company, ...rest } = useLandingCompany();
+
+    const isEuRegion = useMemo(() => {
+        if (!landing_company) return false;
+
+        const eu_shortcode_regex = /^(maltainvest|malta|iom)$/;
+        const eu_excluded_regex = /^mt$/;
+
+        const { gaming_company, financial_company, mt_gaming_company, mt_all_company, id: residence } = landing_company;
+        const financial_shortcode = financial_company?.shortcode;
+        const gaming_shortcode = gaming_company?.shortcode;
+        const mt_gaming_shortcode = mt_gaming_company?.financial?.shortcode || mt_all_company?.swap_free?.shortcode;
+
+        const is_current_mf = financial_shortcode === 'maltainvest';
```

Comment:
Dont think current is the correct wording here.

Reply:
`is_landing_maltainvest` or `is_maltainvest`  ?

Reply:
i think more like is_financial_maltainvest.